toolchain/check/eval.cpp

```diff
@@ -1121,6 +1122,11 @@ auto TryEvalInst(Context& context, SemIR::InstId inst_id, SemIR::Inst inst)
           Phase::Template);
     }
 
+    case CARBON_KIND(SemIR::InstanceConstant instance):
+      // Pull the instance-specific constant value out of the generic instance.
+      return SemIR::GetConstantValueInInstance(
+          context.sem_ir(), instance.instance_id, instance.inst_id);
```

Comment:
```suggestion
    case CARBON_KIND(SemIR::InstanceConstant instance): {
      // Pull the instance-specific constant value out of the generic instance.
      return SemIR::GetConstantValueInInstance(
          context.sem_ir(), instance.instance_id, instance.inst_id);
    }
```

Minor, but while I know this macro compiles correctly without braces when it's one statement, I'd feel better with braces in case more statements get added.